### DISCUSSION TITLE
How to refresh Linewire tables?

### DISCUSSION THREAD

**Discussion by sunilit42** [rappasoft/laravel-livewire-tables#1613] [Q&A] [answered]
Hello,

I created two datatables on same page

on top of page i added button for show hide datatables

Now i need to do refresh datatables when click on show button 

Is there way to do it?

**Comment by lrljoe** [comment, marked as answer]
Believe I may have answered this question in Discord for you, but you just need to dispatch "refreshDatatable" to the relevant table to cause it to refresh.
